FrontEnd/src/components/profileList/ProfileCard.jsx

```diff
@@ -9,12 +9,19 @@ import css from './ProfileCard.module.css';
 import axios from 'axios';
 import CategoryBadges from '../MiniComponents/CategoryBadges';
 import StarForLike from '../MiniComponents/StarForLike';
+import BellForUpdates from '../MiniComponents/BellForUpdates';
 
 const { Paragraph } = Typography;
 
 export default function ProfileCard({ isAuthorized, data }) {
   const { user } = useAuth();
   const [isSaved, setIsSaved] = useState(data.is_saved);
+  const [savedIsUpdated, setSavedIsUpdated] = useState(data.saved_is_updated);
```

Comment:
I'm wondering whether we can pass a callback to update it instead of cleaning it in the state.

Reply:
Please check the new implementation, whether it is correct, or not